File: banchus-common/src/main/java/pe/nanamochi/banchus/domain/dto/MenuContentResponse.java
package pe.nanamochi.banchus.domain.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.util.List;

public record MenuContentResponse(List<MenuImage> images) {
  public record MenuImage(
      String image,
      String url,
      @JsonProperty("IsCurrent") boolean isCurrent,
      String begins,
      String expires) {}
}

File: banchus-app/src/main/kotlin/pe/nanamochi/banchus/infrastructure/controller/dto/MenuContentResponse.kt
package pe.nanamochi.banchus.infrastructure.controller.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class MenuContentResponse(val images: List<MenuImage>) {
    data class MenuImage(
        val image: String,
        val url: String,
        @get:JsonProperty("IsCurrent") val isCurrent: Boolean = true,
        val begins: String? = null,
        val expires: String? = null,
    )
}
